OpenSim/Tools/InverseDynamicsTool.cpp

```diff
@@ -367,13 +367,18 @@ bool InverseDynamicsTool::run()
         SpatialVec equivalentBodyForceAtJoint;
 
         for(int i=0; i<nt; i++){
-            StateVector genForceVec(times[i], nq, &((genForceTraj[i])[0]));
+            StateVector
+                genForceVec(times[i],
+                            SimTK::Vector_<double>(nq,
+                                                   &((genForceTraj[i])[0])));
```

Comment:
`genForceTraj[i]` is already a SimTK::Vector; I think you can pass it directly to the `StateVector` constructor.